core/client/app/templates/setup/three.hbs

```diff
@@ -8,8 +8,7 @@
 
     <form class="gh-flow-invite">
         <label>Enter one email address per line, we’ll handle the rest! <i class="icon-mail"></i></label>
-        {{textarea class="gh-input" name="users" placeholder="john@example.com
-sally.sanders@example.com" value=users}}
+        {{textarea class="gh-input" name="users" placeholder="john@example.com sally.sanders@example.com" value=users}}
```

Comment:
No need to resubmit, just commit --amend and force push :)

On Fri, Jun 19, 2015 at 12:35 PM, jomahoney <notifications@github.com>
wrote:

> In core/client/app/templates/setup/three.hbs
> <https://github.com/TryGhost/Ghost/pull/5455#discussion_r32856607>:
>
> > @@ -8,8 +8,7 @@
> >
> >      <form class="gh-flow-invite">
> >          <label>Enter one email address per line, we’ll handle the rest! <i class="icon-mail"></i></label>
> > -        {{textarea class="gh-input" name="users" placeholder="john@example.com
> > -sally.sanders@example.com" value=users}}
> > +        {{textarea class="gh-input" name="users" placeholder="john@example.com sally.sanders@example.com" value=users}}
>
> Yea that makes sense. I'll resubmit the PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TryGhost/Ghost/pull/5455/files#r32856607>.
>